bitmark-references/security.markdown

```diff
@@ -0,0 +1,73 @@
+# Security feature of the Bitmark Blockchain
+
+## Transactions
+
```

Comment:
For the sections, we really need to say how each section relates to the overall topic (of security).

For this section, we might replace the first three lines with something like:

Individual transactions are secured by cryptographic signatures. Single signatures are used for basic transactions, while countersigned signatures are used for special transfers.